packages/nimble-components/src/table/testing/table.pageobject.ts

```diff
@@ -223,7 +234,7 @@ export class TablePageObject<T extends TableRecord> {
             'nimble-table-group-row'
         );
         return Array.from(groupRows).map((_, i) => {
-            return this.getRenderedGroupHeaderContent(i);
+            return this.getRenderedGroupHeaderTextContent(i);
```

Comment:
I can't put a comment on the actual line, but it seems like this function should be `getAllRenderedGroupHeaderTextContent` rather than `getAllRenderedGroupHeaderContent`.